src/rand.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Authors (alpha order): Adrian Catangiu @acatangiu, Alexandra Lordache @aghecenco, Alexandru Agache @alexandruag, Diana Popa @dianpopa, Matt Forrester @forbesmyester, Radu Weiss @raduweiss
```

Comment:
We don't typically single out individual contributors in the files as this is very easy to miss and is not going to be updated.

Reply:
Sure, my reason was that we'd loose the history as Git cannot track the import. You're right it probably won't be maintained so removal is probably the right call.